### ISSUE TITLE
ArgumentError - Malformed version number string ["5.1", "5.0"]

### ISSUE THREAD

**Issue by ChimpSay** [Lofelt/NiceVibrations#22]
Hi, i am getting this error when running `pod install`.
I have another library that also uses the function `SetBuildProperty` to 5.0, which in NV, `SetBuildProperty` set it to 5.1.

I am wondering if its possible to set it to 5.0 instead?

**Comment by mzzndr** [comment]
Hey @ChimpSay 
If I'm not mistaken the Swift version in the repo is currently set to 5.0, so maybe I'm missing something.

Can you provide more details on your setup? How are you building/importing NV in your project?
What's the other lib that you mention?
Anything that could help us recreate the issue would be appreciated.

**Comment by ChimpSay** [comment]
Hi @mzzndr,

I am importing NV from the Feels package.
The other lib i am using is some custom package made from our publisher.
I'll check out the latest package from Feels again, and see if the swift version is `5.1`

**Comment by ChimpSay** [comment]
Hi @mzzndr 

So i just reimported the Feels package.
And it has NV inside of it. Inside the readme file.

```
Nice Vibrations | Haptic Feedback for Mobile
v4.1.2, using Lofelt Studio SDK v1.3.4

This Asset Package includes everything you need to begin adding advanced haptic feedback to your mobile applications created with Unity. It consists of a few important parts:

- The Nice Vibrations plug-in for Unity: Provides haptic playback functionality, as a code-only API for C# scripts and as a MonoBehaviour API for the Unity editor.

- Haptic Samples: To help you get started fast without needing to design any haptics first, the Asset includes a growing collection of free-to-use haptic clips with associated audio files covering use cases such as Application UX and game effects. You can drop these into your app and start experiencing haptics on your mobile device immediately.

- Demo: This is a project you can build for either iOS (iPhone) or Android which showcases the current functionality of this package.

- Older Versions: Older versions of Nice Vibrations are included as well, for reference and as a porting aid. You can run the current version side-by-side with the older 3.9 version in case you haven't ported everything to the new API yet.

Full documentation can be found at https://github.com/Lofelt/NiceVibrations/wiki.

Nice Vibrations includes the Lofelt Studio SDK, which uses third-party libraries. See 3RD-PARTY-LICENSES.md for a list of libraries and their licenses.
```
So i would assume its version `4.1.2`. 
There is a file called `MMNVBuildPostProcessor.cs` that sets the swift version to 5.1.

**Comment by reunono** [comment]
Hi @ChimpSay, the version of NV included in Feel is indeed v4.1.2.
There is no MMNVBuildPostProcessor.cs if you install Feel though, the only place where this file could be found is if you install a previous version (found in the packages in the OlderVersions folder).
Unless you unpack one of these, or unless you had a previous install already in your project and didn't delete it, there shouldn't be a MMNVBuildPostProcessor.

**Comment by ChimpSay** [comment]
Ah. That would make sense then.
So its safe to remove that file then i would assume.
